eclipse/che
# Include JSON schema for devfile 2.1.0-alpha

### What does this PR do?
Includes JSON schema for devfile 2.1.0-alpha in Che Server in order to use devfiles with this schema version.

### Screenshot/screencast of this PR



### What issues does this PR fix or reference?

Che Server should include JSON schema for devfile 2.1.0-alpha #19304

### How to test this PR?

1. Deploy Che on Openshift with this che-server image (quay.io/pbaran/che-server:schema) or change the image in running deployment.
2. Enable DevWorkspaces
3. Probe the /devfile endpoint with schema version setting: 2.1.0-alpha
-- Schema version 2.1.0-alpha should be returned
4. Create a workspace from devfile with schema version: 2.1.0-alpha
-- Workspace should be created

### Current problems with creating workspace
When creating workspace through factory from devfile 2.1.0-alpha (https://github.com/xbaran4/Test/blob/master/devfile.yaml) this error occurs.
![image](https://user-images.githubusercontent.com/73115616/114161117-fd74da00-9927-11eb-8f24-2cee22f75bae.png)
Solved!
### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.
